src/fluree/db/connection.cljc

```diff
@@ -450,7 +443,7 @@
       (let [alias (if (fluree-address? alias)
                     (nameservice/address-path alias)
```

Comment:
Now that I think about it, we should call `ensure-ledger-branch` on this branch as well, I don't know that an address is guaranteed to have a branch component.